plugins/arDominionB5Plugin/js/multiFileUpload.js

```diff
@@ -4,7 +4,8 @@
 
   function MultiFileUpload (element)
   {
-    this.$element = $(element).find('div.multiFileUpload');
+    this.$multiFileUpload = $(element).find('div.multiFileUpload');
+    this.$element = $(element);
```

Comment:
I don't understand why do you need two vars here ... based on line 386, you already have the correct element and line 8 should be enough to use `this.$element` for everything ... right?

Reply:
I have fixed this underlying issue allowing me to remove the second variable.